pyproject.toml

```diff
@@ -25,17 +25,17 @@ python = "^3.6"
 requests = "^2.22"
 simplejson = "^3.16"
 dataclasses = "^0.6.0"
+pandas = "^0.25.3"
```

Comment:
This is a pretty heavyweight change.
In general, pandas is not *required* to use python-client; it's optional, only required if you use dataset.collection.create_from_dataframe()

Reply:
It was (and remains) a `dev` dependency, so this only affects people developing Python Client that need to run tests for `create_from_dataframe()`. This does not require `pandas` for users that install without dev deps (e.g. from PyPI)